plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveTableProperties.java

```diff
@@ -240,6 +255,13 @@ public static BucketingVersion getBucketingVersion(Map<String, Object> tableProp
         throw new TrinoException(INVALID_TABLE_PROPERTY, format("%s must be between 1 and 2 (inclusive): %s", BUCKETING_VERSION, property));
     }
 
+    @SuppressWarnings("unchecked")
+    public static List<SortingColumn> getTrinoSortBy(Map<String, Object> tableProperties)
```

Comment:
since we keep these sort properties, we should return them to engine (both for `sorted_by` and `sort_by`). I know engine doesn't utilize these at the moment, but it will be improved

Reply:
To the engine as what? `ConnectorTableMetadata`?

since they cannot be guaranteed, we can consider them "optimizer hints", but the plan's _correctness_ must not depend on them

Reply:
Good point. We cannot return properties here.